src/aws/osml/model_runner/inference/feature_utils.py

```diff
@@ -16,6 +16,9 @@
 from aws.osml.model_runner.common import ImageDimensions
 from aws.osml.photogrammetry import GeodeticWorldCoordinate, SensorModel
 
+# Enable exceptions for GDAL
+gdal.UseExceptions()
+
```

Comment:
This is a global setting for the entire application; do we really want to bury it down in some low level utility module?

Reply:
It is set at a higher level, in ``app.py``, and won't actually be needed in the next version of GDAL. But as to why this was put in, we import this module/file directly in our unit tests and it causes us to throw a warning because it isn't set locally.

Reply:
Could this go in the unit test code instead if it is just to address warning encountered in them?

Reply:
I will remove it